--- cffi-1.11.5/testing/cffi1/test_zdist.py.orig	2018-02-27 19:15:41.000000000 +0100
+++ cffi-1.11.5/testing/cffi1/test_zdist.py	2018-05-27 11:40:42.977033830 +0200
@@ -17,8 +17,8 @@
 
     def setup_method(self, meth):
         self.executable = os.path.abspath(sys.executable)
-        self.rootdir = os.path.abspath(os.path.dirname(os.path.dirname(
-            cffi.__file__)))
+        self.rootdir = os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(
+            cffi.__file__))))
         self.udir = udir.join(meth.__name__)
         os.mkdir(str(self.udir))
         if meth.chdir_to_tmp:
--- cffi-1.11.5/testing/embedding/test_basic.py.orig	2018-02-27 19:15:41.000000000 +0100
+++ cffi-1.11.5/testing/embedding/test_basic.py	2018-05-27 11:41:38.247033196 +0200
@@ -26,7 +26,7 @@
 
 
 def prefix_pythonpath():
-    cffi_base = os.path.dirname(os.path.dirname(local_dir))
+    cffi_base = os.path.dirname(os.path.dirname(os.path.dirname(local_dir)))
     pythonpath = org_env.get('PYTHONPATH', '').split(os.pathsep)
     if cffi_base not in pythonpath:
         pythonpath.insert(0, cffi_base)
